src/Command/NewPage.php

```diff
@@ -11,69 +9,89 @@
 namespace Cecil\Command;
 
 use Cecil\Util\Plateform;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputDefinition;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\ConfirmationQuestion;
 use Symfony\Component\Process\Process;
-use Zend\Console\Prompt\Confirm;
 
-/**
- * Class NewPage.
- */
-class NewPage extends AbstractCommand
+class NewPage extends Command
 {
     /**
-     * @var string
-     */
-    protected $name;
-    /**
-     * @var bool
+     * {@inheritdoc}
      */
-    protected $force;
+    protected function configure()
+    {
+        $this
+            ->setName('new:page')
+            ->setDescription('Create a new page')
+            ->setDefinition(
+                new InputDefinition([
+                    new InputArgument('name', InputArgument::REQUIRED, 'New page name'),
+                    new InputArgument('path', InputArgument::OPTIONAL, 'If specified, use the given path as working directory'),
+                    new InputOption('force', 'f', InputOption::VALUE_NONE, 'Override the file if already exist'),
+                    new InputOption('open', 'o', InputOption::VALUE_NONE, 'Open editor automatically'),
+                ])
+            )
+            ->setHelp('Create a new page file (with a default title and the current date).');
+    }
 
     /**
      * {@inheritdoc}
      */
-    public function processCommand()
+    protected function execute(InputInterface $input, OutputInterface $output)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method execute() has a Cyclomatic Complexity of 10. The configured cyclomatic complexity threshold is 10.](https://app.codacy.com/manual/Narno/Cecil/pullRequest?prid=4722362)